apps/emqx_management/src/emqx_mgmt_cli.erl

```diff
@@ -601,21 +602,13 @@ listeners(_) ->
 %% @doc authz Command
 
 authz(["cache-clean", "node", Node]) ->
-    case emqx_mgmt:clean_authz_cache_all(erlang:list_to_existing_atom(Node)) of
-        ok ->
-            emqx_ctl:print("Authorization cache drain started on node ~ts.~n", [Node]);
```

Comment:
the words were carefully picked.
this is only a timestamp flag set, so the clients will start draining their ACL cache

the new log made it look like the cache is drained immediately